api/src/pcapi/routes/pro/venues.py

```diff
@@ -79,8 +79,11 @@ def get_venues(query: venues_serialize.VenueListQueryModel) -> venues_serialize.
         offerer_id=offerer_id,
         validated_offerer=query.validated,
     )
+    venues_with_offers = (
+        offerers_repository.get_venues_with_offers(venue_list[0].managingOffererId) if len(venue_list) > 0 else None
```

Comment:
Cf. mon commentaire dans `VenueListItemResponseModel.from_orm()` : si on fait en sorte de passer une liste (potentiellement vide) mais jamais None, cela simplifiera le code : 

```suggestion
        offerers_repository.get_venues_with_offers(venue_list[0].managingOffererId) if venue_list > 0 else []
```

(Au passage, j'ai simplifié l'expression de la condition, ce sera plus "pythonique".)

Et en parlant de typage, `get_venues_with_offers()` est mal typé, elle ne peut pas renvoyer `None` : 

```diff
- def get_venues_with_offers(offererId: int) -> Iterable[int] | None:
+ def get_venues_with_offers(offererId: int) -> Iterable[int]:
```

Reply:
@dbaty Avec les suggestions, je me retrouve avec l'erreur `error: Unsupported operand types for > ("List[Venue]" and "int")`